### ISSUE TITLE
+1 on tab and favicon doesn't clear on focus, but blur

### ISSUE THREAD

**Issue by fitzgen** [fitzgen/bugzilla-todos#14]
I would assume that if I select the tab with the new activity, then the indicator would go away, but it doesn't go away until you focus a different tab after focusing that one.

**Comment by harthur** [comment]
Agreed, but I'd want the glowy yellow highlight around the new items to stay until blur.
